pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/admin/AddUserRequest.java

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.dsm.service.admin;
+
+import java.util.List;
+
+import lombok.Data;
+import org.broadinstitute.dsm.db.dto.user.UserDto;
+
+@Data
+public class AddUserRequest {
+
+    private final List<AddUserRequest.User> users;
+
+    public AddUserRequest(List<AddUserRequest.User> users) {
+        this.users = users;
+    }
+
+    @Data
+    public static class User {
+        private final String email;
+        private final String name;
+        private final String phone;
+        private final List<String> roles;
+
+        public User(String email, String name, String phone, List<String> roles) {
```

Comment:
We can use @AllArgsConstructor and get rid of this constructor

Reply:
looks clean and less code, thats all.

Reply:
Thx. You may have noticed that I used it for the updated UserDto. It works well there.